### ISSUE TITLE
Quality and JSON structure have changed

### ISSUE THREAD

**Issue by pharaofranz** [leogaggl/media#4]
I just noticed that the ARD seems to have changed their JSON structures. There is no Quality=3 anymore (highest is 2, seems to be former 3) and there are two files in the '.stream' output (not a JSON-pro... I'd call it a dictionary-key). The file you want for highest quality is the second entry. Thus, I changed the default QUALITY to 2 and in the DOWNLOADURL in the last jq-call: '.stream[1]'.

Cheers,
Franz

**Comment by dicer** [comment]
Yep, I just changed that in auto-tatort as well. Here is what quality I found for which stream:

0.0 320x180 (53k audio)
1.0 512x288 (93k audio)
1.1 480x270 (61k audio)
2.0 640x360 (189k audio)
2.1 960x540 (189k audio)
